src/test/java/org/candlepin/model/EnvironmentTest.java

```diff
@@ -42,4 +47,52 @@ public void testGetOwnerKeyWithNoOwner() {
         assertNull(environment.getOwnerKey());
     }
 
+    @Test
+    public void testSetContentPrefix() {
+        Environment environment = new Environment();
+        Date before = minusTwoSeconds();
+        environment.setLastContentUpdate(before);
+
+        assertNull(environment.getContentPrefix());
+
+        environment.setContentPrefix("newPrefix");
+        Date after = environment.getLastContentUpdate();
+
+        assertEquals("newPrefix", environment.getContentPrefix());
+        assertTrue(after.after(before));
+    }
+
+    @Test
+    public void testSetLastContentUpdate() {
```

Comment:
This should be renamed to something indicative of verifying the UUT rejects a null input

Reply:
done, also split the test for more clarity